how-to/scale/with-tiup.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Scale the TiDB Cluster Using TiUP
+summary:
+Category: how-to
```

Comment:
summary: xxx
category: how-to
aliases: ['/docs/stable/how-to/scale/with-tiup/']

Reply:
@ran-huang Please fix the meta info after the doc special week.

Reply:
OK